### ISSUE TITLE
Batch size implementation missing in load_data.py

### ISSUE THREAD

**Issue by aplusc98** [prakashpandey9/BicycleGAN#3]
Although we can pass a batch size argument in the main.py code In the load_data.py there is no implementation to fetch the respective batches..rather it always fetches a batch size of 1. Hence leading to this error.

Cannot feed value of shape (1, 256, 256, 3) for Tensor u'input_images:0', which has shape '(2, 256, 256, 3)'

Whenever a non-default batch is taken.

**Comment by prakashpandey9** [comment]
@aplusc98  Yes, the implementation is only for batch_size = 1. You can only pass the index of a single image as an argument to the `load_batch_image` function in `load_data.py`. I will update the code for any batch_size very soon.
